_projects/2022/100383017/100383017.Rmd

```diff
@@ -0,0 +1,634 @@
+plot
+```
+
+Now its time to work on the appareance of our graph. The panel grid lines will be removed by setting an element_blank() argument and the aspect ratio will be set so that we can observe a wider spiral in the final graph. The colors chosen for the bar plots are green and purple which contrast among then whereas the background layers will be set in black as a final step:
+
+```{r}
+plot <- plot +
+scale_fill_discrete(type = c("purple", "green"))+
+  theme(legend.background = element_rect(color = "black"),
+        plot.title = element_text(hjust = 0.5, vjust = 3, size =16),
+        panel.background = element_rect(fill = "white"),
+        panel.grid.major.x = element_blank(),
+        panel.grid.major.y = element_blank(),
+        aspect.ratio = 0.8,
+        plot.subtitle = element_text(size = 9,hjust = 0),
+        plot.caption = element_text(size = 9,hjust = 0),
+        legend.key = element_rect(color = "white", size = 2),
+        legend.key.width = unit(2,"lines"),
+        legend.title = element_text(hjust = 0.5))
+plot
+```
+
+We will have to add some annotations in our plot to point the amount of growth injection each economy needs. These lines and annotations will make more sense once the polar coordinates have been applied. We encourage the reader to come back to these steps after the plot is over to understand how each of our elements gets settled in the final spiral.
+
+```{r}
+plot <- plot +
+geom_abline(slope = 0,
+            intercept = c(-2,-1,0,1,2,3,4),
+            color = rgb(0,0,0,0.3), linetype = "dotted")+
+  annotate(geom="text", x=-10,
+           y=c(-2,-1,0,1,2,3,4), label=c("-2%","-1%","0%","1%","2%","3%","4%"),
+           color="black",size =3,angle = 28)
+plot
+```
+
+Finally, we have to add some rectangular geometries for different segments of the plot which will be given some alpha (transparency) to account for best to worse layers. These layers mimic the level curves of a three dimensional bowl in which each layer projected in the floor corresponds to a circle. We will see this once we apply our polar coordinates.
+
+```{r}
+plot <- plot +
+annotate("rect", ymin = -2, ymax = -1,
+         xmin = -10, xmax = 110, fill = "black", alpha = 0.1)+
+  annotate("rect", ymin = -1, ymax = 0,
+           xmin = -10, xmax = 110, fill = "black", alpha = 0.2)+
+  annotate("rect", ymin = 0, ymax = 1,
+           xmin = -10, xmax = 110, fill = "black", alpha = 0.3)+
+  annotate("rect", ymin = 1, ymax = 2,
+           xmin = -10, xmax = 110, fill = "black", alpha = 0.4)+
+  annotate("rect", ymin = 2, ymax = 3,
+           xmin = -10, xmax = 110, fill = "black", alpha = 0.5)+
+  annotate("rect", ymin = 3, ymax = 4,
+           xmin = -10, xmax = 110, fill = "black", alpha = 0.6)
+plot
+```
+
+We can now apply our polar coordinates!
+
+
+```{r}
+plot <- plot +
+coord_polar()
+plot
+```
+
+
+This is the end of the project
```

Comment:
This is a bit blunt. :) Please, add some concluding remarks.

Reply:
Hello teacher,

I have accounted for the issues you mentioned. However, when I try to push the Rmd and HTML again, i get another error
<img width="722" alt="Captura de pantalla 2023-01-03 133259" src="https://user-images.githubusercontent.com/118019450/210358123-21d0d428-0ecd-454d-9c63-fdc67a5a9c86.png">

What can I do? 
Thank you!!

Reply:
Probably because you removed the file from the GitHub interface, right? This means another commit on GitHub that is not in your local copy. You need to pull that commit before pushing anything.

Reply:
Sorry, I have been trying for a lot of time but I honestly don´t understand what I should do here. 

Thank you in advanced, Nacho

Reply:
I mean, in the git panel, in RStudio, you have a "Push" button, but also a "Pull" button, right? Click "Pull" to download the changes you made in GitHub (removing the CSV) to your local repository. Then you should be able to push again.

Reply:
Or, wait, let me try to undo your last commit.

Reply:
Done. Try to push again.

Reply:
Thank you very much, Im so sorry, I am very new to github... 
Shall I do anything else? Otherwise, I will continue working on the powerpoint presentation for next week. 

Thank you again! Nacho